farmsmart/farmsmart-firebase
# Feature/use new template

# JIRA
Use new scoring template for user ease of use and reduce manual errors


# DETAILS
### :construction: Change
Added a new cloud function which will process the new template in the SCORE MATRIX sheet.

### :traffic_light: Testing
1. Update the recommendations spreadsheet
2. Invoke the cloud function
3. Check the firestore fs_crop_scores collection for data read.

### :heavy_check_mark: Coding Standard Checklist

- [X] Smoke Test done or deployed on Firebase
